frappe/desk/desktop.py

```diff
@@ -127,6 +127,8 @@ def is_item_allowed(self, name, item_type):
 			return name in self.allowed_reports
 		if item_type == "help":
 			return True
+		if item_type == "dashboard":
+			return True
```

Comment:
@prssanna will this be fine? I presume only permitted docs will show up

Reply:
Yup only permitted charts will show up on the dashboard